doc/TILESET.md

```diff
@@ -1,141 +1,191 @@
-This is a Python script that will convert a legacy tileset into a compositing tileset.  It reads the `tile_config.json` and assigns semi-arbitrary file names to each sprite index.  Then it changes all the sprite indexes references to the file names.  Then it breaks up `tile_config.json` into many small tile_entry JSON files with arbitrary file names, and pulls out each sprite and writes it to aseparate file.
+##### Sprite 
+A single image that represents either an individual game entity or a common background
 
-It requires pyvips to do the image processing.
+##### Root name
+File name without an extension.
 
-It takes a single mandatory argument, which is the path to the tileset directory.  For example:
-`python3 tools/gfx_tools/decompose.py gfx/ChestHole16Tileset` will convert the legacy ChestHole16 tileset to a compositing tileset.
+##### Tile
+A unit square in the game world with certain coordinates, where entities can exist and on which sprites are drawn. Sometimes used (incorrectly) in place of "sprite".
 
-decompose.py creates a sufficient directory hierarchy and file names for a tileset to be compositing, but it is machine generated and badly organized.  New compositing tilesets should use more sensible file names and a better organization.
+##### Tilesheet
+A collection of sprites with identical sizes and offsets composited into one image file that the game will read.
 
-It shouldn't be necessary to run decompose.py very often.  Legacy tilesets should only need to be converted to composite tilesets one time.
+##### `tile_config.json` file
+Machine-readable description mapping the contents of tilesheets to game entities.
 
-### tools/gfx_tools/compose.py
-This is a Python script that creates the tilesheets for a compositing tileset.  It reads all of the directories in a tileset's directory with names that start with `pngs_` for sprite files and `tile_entry` JSON files, creates mappings of sprite file names to indices, merges the sprite files into tilesheets, changes all of the sprite file name references in the `tile_entries` to indices, and merges the `tile_entries` into a `tile_config.json`.
+##### `tileset.txt` file
+A collection of tileset metadata.
 
-Like decompose.py, it requires pyvips to the image processing.
+##### Tile entry
+JSON configuration object that describes which sprites are used for which game entities and how.
 
-The original sprite files and `tile_entry` JSON files are preserved.
+##### Compositing Tileset
+A tileset that is stored as directories of individual sprites and tile entries.
 
-### directory structure
-Each compositing tileset has one or more directories in it with a name that starts with `pngs_`, such as `pngs_tree_32x40` or `pngs_overlay`.  These are the image directories.  All of the sprites in an image directory must have the same height and width and will be merged into a single tilesheet.
+##### `compose.py` script
+This creates a package suitable for the game from a compositing tileset.
```

Comment:
```suggestion
This creates a package suitable for the game (in tilesheet form) from a compositing tileset.
```
Or something like that - otherwise, it's unclear exactly what "package suitable for the game" means.

Reply:
Thank you. I tried rewriting it and I guess I forgot one article, please verify.